doc/how_to_guides/pilz_industrial_motion_planner/src/pilz_sequence.cpp

```diff
@@ -0,0 +1,366 @@
+auto const LOGGER = rclcpp::get_logger("pilz_sequence_node");
+
+int main(int argc, char** argv)
+{
+  // Initialize ROS and create the Node
+  rclcpp::init(argc, argv);
+  rclcpp::NodeOptions node_options;
+  node_options.automatically_declare_parameters_from_overrides(true);
+
+  auto node = rclcpp::Node::make_shared("pilz_sequence_node", node_options);
+
+  // We spin up a SingleThreadedExecutor so MoveItVisualTools interact with ROS
+  rclcpp::executors::SingleThreadedExecutor executor;
+  executor.add_node(node);
+  std::thread([&executor]() { executor.spin(); }).detach();
+
+  // Planning group
+  static const std::string PLANNING_GROUP = "panda_arm";
+
+  // Create the MoveIt MoveGroup Interface
+  // In this case, this is just necessary for the visual interface
+  using moveit::planning_interface::MoveGroupInterface;
+  auto move_group_interface = MoveGroupInterface(node, PLANNING_GROUP);
+
+  // Construct and initialize MoveItVisualTools
+  auto moveit_visual_tools = moveit_visual_tools::MoveItVisualTools{ node, "panda_link0", "move_group_tutorial",
+                                                                     move_group_interface.getRobotModel() };
+  moveit_visual_tools.deleteAllMarkers();
+  moveit_visual_tools.loadRemoteControl();
+  moveit_visual_tools.prompt("Press 'next' in the RvizVisualToolsGui window to continue the demo");
+
+  // [ --------------------------------------------------------------- ]
+  // [ ----------------------- Motion Sequence ----------------------- ]
+  // [ --------------------------------------------------------------- ]
+
+  // ----- Motion Sequence Item 1
+  // Create a MotionSequenceItem
+  moveit_msgs::msg::MotionSequenceItem item1;
+
+  // Set pose blend radius
+  item1.blend_radius = 0.1;
+
+  // MotionSequenceItem configuration
+  item1.req.group_name = PLANNING_GROUP;
+  item1.req.planner_id = "LIN";
+  item1.req.allowed_planning_time = 5;
+  item1.req.max_velocity_scaling_factor = 0.1;
+  item1.req.max_acceleration_scaling_factor = 0.1;
+
+  moveit_msgs::msg::Constraints constraints_item1;
+  moveit_msgs::msg::PositionConstraint pos_constraint_item1;
+  pos_constraint_item1.header.frame_id = "world";
+  pos_constraint_item1.link_name = "panda_hand";
+
+  // Set a constraint pose
+  auto target_pose_item1 = [] {
+    geometry_msgs::msg::PoseStamped msg;
+    msg.header.frame_id = "world";
+    msg.pose.position.x = 0.3;
+    msg.pose.position.y = -0.2;
+    msg.pose.position.z = 0.6;
+    msg.pose.orientation.x = 1.0;
+    msg.pose.orientation.y = 0;
+    msg.pose.orientation.z = 0;
+    msg.pose.orientation.w = 0;
+    return msg;
+  }();
+  item1.req.goal_constraints.push_back(
+      kinematic_constraints::constructGoalConstraints("panda_link8", target_pose_item1));
+
+  // ----- Motion Sequence Item 2
+  // Create a MotionSequenceItem
+  moveit_msgs::msg::MotionSequenceItem item2;
+
+  // Set pose blend radius
+  // For the last pose, it must be 0!
+  item2.blend_radius = 0;
```

Comment:
nit: For all these values, in this file and the docs, could you replace the `0` with `0.0`?

And same with the `5` in the planning time.